src/Pages/Home.js

```diff
@@ -1,12 +1,19 @@
 import React from "react";
 import '../Styling/Home.css';
+import NavBar from '../Components/NavBar'
+import SocialIconsEmailPlacement from "../Components/SocialIconsEmailPlacement";
 
 function Home(){
     return(
         <div className="Home">
+            <div className="NavBar_Container">
+                <NavBar />
```

Comment:
Let’s discuss how we can add Navbar globally regardless of what route you’re on, in our next meeting. Adding the Navbar component manually in each page it’s used is redundant and unnecessary